test/jasmine/tests/funnelarea_test.js

```diff
@@ -1227,26 +1241,31 @@ describe('Test event data of interactions on a funnelarea plot:', function() {
         it('should not emit a hover if you\'re dragging', function() {
             gd._dragging = true;
             mouseEvent('mouseover', pointPos[0], pointPos[1]);
-            expect(futureData).toBeUndefined();
+            expect(futureData).toBe(null);
         });
 
         it('should not emit a hover if hover is disabled', function() {
-            Plotly.relayout(gd, 'hovermode', false);
-            mouseEvent('mouseover', pointPos[0], pointPos[1]);
-            expect(futureData).toBeUndefined();
+            Plotly.relayout(gd, 'hovermode', false)
+            .then(function() {
+                mouseEvent('mouseover', pointPos[0], pointPos[1]);
+                expect(futureData).toBe(null);
+            });
```

Comment:
`done`/`failTest`

Reply:
Good eye. Fixed in https://github.com/plotly/plotly.js/pull/5340/commits/2c0b913359024b733f76849a2e45f32842f2fd0e.